verify.ts

```diff
@@ -0,0 +1,265 @@
+const commandLineArgs = require('command-line-args');
```

Comment:
other scripts uses `yargs` already

Reply:
I understand the desire to have only one system here, though it is not a huge deal.  I think `command-line-args` has superior output and would use it instead of yargs.

Reply:
I took another look at this and find that migrate.js is using functionality from yargs that is not available in command-line-args, but the yargs funtionality comes at the cost of ugly and difficult-to-parse output.  The verify code doesn't need the extra functionality of yargs, and by using command-line-args has much nicer output.  So I am leaving well-enough alone here.